src/Application/Regulation/Query/GetRegulationsQueryHandler.php

```diff
@@ -32,16 +32,21 @@ public function __invoke(GetRegulationsQuery $query): Pagination
         );
         $regulationOrderViews = [];
 
-        foreach ($regulations as $regulation) {
+        foreach ($regulationOrderRecords as $regulationOrderRecord) {
+            $regulationOrder = $regulationOrderRecord->getRegulationOrder();
+            $locations = $regulationOrder->getLocations();
+            $nbLocations = $locations->count();
+
             $regulationOrderViews[] = new RegulationOrderListItemView(
-                $regulation['uuid'],
-                $regulation['identifier'],
-                $regulation['status'],
-                $regulation['address'] ? new LocationView(
-                    LocationAddress::fromString($regulation['address']),
+                uuid: $regulationOrderRecord->getUuid(),
+                identifier: $regulationOrder->getIdentifier(),
+                status: $regulationOrderRecord->getStatus(),
+                numLocations: $nbLocations > 1 ? $nbLocations - 1 : 0,
```

Comment:
Attention, je dirais que soit on l'appelle `numLocations` et dans ce cas ça vaut 2 s'il y a 2 localisations au total, soit on l'appelle comme avant et dans ce cas ça vaut 1 s'il y a 2 localisations au total. Mais ici on mélange les deux

```suggestion
                numLocations: $nbLocations,
```